### DISCUSSION TITLE
Help Needed with Logo Display Issue After Merge

### DISCUSSION THREAD

**Discussion by YakeDev** [Ctoic/Lisbook#84] [General]
Hi team,

I'm experiencing an issue with the logo display on our site after merging the latest changes.

**Here's what's happening:**

- Locally, the logo displays perfectly as expected:

<img width="1288" alt="image" src="https://github.com/user-attachments/assets/2c84239a-63f6-4717-9ed3-a1042866adb5">

- Locally, the logo displays perfectly as expected:

<img width="1171" alt="image" src="https://github.com/user-attachments/assets/e6063037-ec8c-46e8-8443-0e1f87bc5c46">

However, after the merge, when I check the site online, the logo looks broken/misaligned:

I've double-checked the file path, and everything seems to be in order. It works fine locally, so I'm not sure what's causing the issue on the live environment.

Could you help me troubleshoot this? Any insights on why the logo might not be displaying correctly after the merge would be greatly appreciated.

Thanks in advance!

@Ctoic

**Comment by YakeDev** [comment]
@Ctoic

**Comment by Ctoic** [comment]
Working on it similar issue I had faced in the past , It was related to the path we provide. Some times it is because of multiple js files we add and they contradict with each other. But I'm on it to find the solotion.

**Reply by YakeDev** [reply]
Okay, nice.
